zerver/openapi/test_curl_examples.py

```diff
@@ -117,8 +114,15 @@ def test_generated_curl_examples_for_success(client: Client, owner_client: Clien
                     )
                     raise
 
-    if REGISTERED_GENERATOR_FUNCTIONS != CALLED_GENERATOR_FUNCTIONS:
+    if (
+        curl_global_variable.REGISTERED_GENERATOR_FUNCTIONS
+        != curl_global_variable.CALLED_GENERATOR_FUNCTIONS
+    ):
         raise Exception(
             "Some registered generator functions were not called:\n"
-            " " + str(REGISTERED_GENERATOR_FUNCTIONS - CALLED_GENERATOR_FUNCTIONS)
+            " "
+            + str(
+                curl_global_variable.REGISTERED_GENERATOR_FUNCTIONS
+                - curl_global_variable.CALLED_GENERATOR_FUNCTIONS
+            )
```

Comment:
The right way to do this is just to export a function from the `curl_param_value_generators`  module that returns a list of registered functions that weren't called.  That would be a good readability cleanup refactor regardless of the problem you were seeing, so let's just do that for this first commit.

That said, generally importing a list (rather than an integer) from another module should work fine, since you get a pointer to the original mutable object.

Reply:
@timabbott Having a function for this sounds great. I'll refactor this.
About importing the list, I initially did this, but after some suggestion [here](https://github.com/zulip/zulip/pull/17870#pullrequestreview-625619989) and I also think that maintaining a list with just one element in it doesn't sound very good. 
Anyways, it solves the issue, so should I make it list again?